armi/utils/pathTools.py

```diff
@@ -54,29 +54,34 @@ def armiAbsPath(*pathParts):
         return result
 
 
-def copyOrWarn(fileDescription, sourcePath, destinationPath):
-    """Copy a file, or warn if the file doesn't exist.
+def copyOrWarn(filepathDescription, sourcePath, destinationPath):
+    """Copy a file or directory, or warn if the filepath doesn't exist.
 
     Parameters
     ----------
-    fileDescription : str
+    filepathDescription : str
         a description of the file and/or operation being performed.
     sourcePath : str
-        Path of the file to be copied.
+        Filepath to be copied.
     destinationPath : str
-        Path for the copied file.
+        Copied filepath.
     """
     try:
-        shutil.copy(sourcePath, destinationPath)
+        if os.path.isdir(sourcePath):
+            shutil.copytree(sourcePath, destinationPath)
+        else:
+            shutil.copy(sourcePath, destinationPath)
         runLog.debug(
-            "Copied {}: {} -> {}".format(fileDescription, sourcePath, destinationPath)
+            "Copied {}: {} -> {}".format(
+                filepathDescription, sourcePath, destinationPath
+            )
         )
     except shutil.SameFileError:
         pass
```

Comment:
this exception is untestable. it WORKS properly but if I try to `assertRaises(shutil.SameFileError)` then I get this failure:

![image](https://github.com/user-attachments/assets/d5bdd95a-6e8a-40d9-ae2f-e5b86cea2147)

Reply:
Yeah, it would be hard to test the `pass` line. They usually are.

In our code coverage calculation, ARMI actually ignores `pass` lines, and does not count them.

Thanks for trying!  But you're good.